### DISCUSSION TITLE
Zmizet Právě jedoucí

### DISCUSSION THREAD

**Discussion by rblaha15** [jaro-jaro/DPMCB#143] [Ideas]
Navrhuju aby karta `Právě jedoucí` zmizela vždy když budeš mít datum jiný než dnes

Tímto by se celá aplikace chovala konzistentně vůči jiným datům - a to je žádné online info

**Comment by 0ndra0** [comment]
jo, to neni špatnej nápad

**Comment by rblaha15** [comment]
- \>\>\> #153
